dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM centos:7
+RUN yum -y install rpm-build epel-release
+RUN yum -y install python36 python36-devel git make gcc gcc-devel automak libtool make
```

Comment:
I believe `automak` should be `automake` -- but do we really need this, or `gcc` or `libtool`? Done't see anywhere we're using this stuff.